tasks/handle-action-check.yaml

```diff
@@ -1,39 +1,29 @@
 ---
-- when: >-
+- name: Check status of Provision
+  when: >-
     'completeTimestamp' not in vars.anarchy_subject.status.towerJobs.provision
   block:
-  - name: >-
-      {% if 'deployerJob' in vars.anarchy_subject.status.towerJobs.provision %}
-      Get provision deployer job for {{ anarchy_subject_name }}
-      {% else %}
-      Get provision launch job for {{ anarchy_subject_name }}
-      {% endif %}
+  - name: Get provision deployer job for {{ anarchy_subject_name }}
     vars:
-      # Check the deployer job if present, otherwise check launch job
-      _provision_tower_jobs: >-
-        {{ vars.anarchy_subject.status.towerJobs.provision | default({}) }}
-      _job_id: >-
-        {{ _provision_tower_jobs.deployerJob
-         | default(_provision_tower_jobs.launchJob)
-         | default('')
-        }}
+      __job_id: "{{ vars.anarchy_subject.status.towerJobs.provision.deployerJob }}"
+    # The awx.awx collection does not have a tower_job_info module
     uri:
-      url: https://{{ babylon_tower.hostname }}/api/v2/jobs/{{ _job_id }}/
+      url: https://{{ babylon_tower.hostname }}/api/v2/jobs/{{ __job_id }}/
       url_username: "{{ babylon_tower.user }}"
       url_password: "{{ babylon_tower.password }}"
       force_basic_auth: true
       # FIXME - Should validate certs
       validate_certs: false
       return_content: true
     register: r_api_response
-    # FIXME - Handle API errors
-    #ignore_errors: true
+    retries: 6
+    delay: 10
```

Comment:
variables with good default for 6 and 10 ?

Reply:
Good catch... actually we have no `until` along with no `retries` and `delay`. The `uri` module has some built-in retry logic, and the thought was to catch when tower is temporarily down... but Anarchy will just retry anyway. So probably no reason for the retry.